### PR TITLE
update auth token class methods

### PR DESCRIPTION
**Pull request by festinalente91**
- changed singleton class AuthToken to use static methods

### CODE REVIEW

File: src/server/auth-token.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ class AuthToken {
 
   constructor() {}
 
-  encode(tokenType, payload) {
+  static encode(tokenType, payload) {
```
Comment (ochanje210) [inline src/server/auth-token.js:34]: how does this make difference?

Comment (hewonjeong) [reply src/server/auth-token.js:34]: 22

Comment (festinalente91) [reply src/server/auth-token.js:34]: This class was using singleton pattern, and in the authenticate method, there is a "this" which is the context(object). So when I use it in the index.js like AuthToken.authenticate, this method have no context, which means it cannot use this.decode method (this is undefined).
I solved it using .bind() in the index.js, but I thought this solution is much more simpler and safe solution for now.

Comment (ochanje210) [reply src/server/auth-token.js:34]: great. I believe in your decision.

**Comment by ochanje210** [comment]
lgtm

**Comment by hewonjeong** [comment]
Lgtm

**Pull request merged**